### DISCUSSION TITLE
How to use soft break in JUSTIFIED alignment in last line of paragraph

### DISCUSSION THREAD

**Discussion by bkmalan** [dolanmiu/docx#1033] [Q&A]
Hi,
First of all, Thanks to the contributors for maintaining this great library.

Now coming to my concern, I'm using JUSTIFIED as my alignment type but few paragraphs' last line doesn't cover the entire width so it is justified in this manner.
![Screenshot from 2021-07-08 14-11-45](https://user-images.githubusercontent.com/55806149/124891481-a77f0380-dff6-11eb-87c7-b279ac34bf23.png)

But I don't want the last line to be justified as much as the previous lines.
I went through the documentation and found this line `this.doc.Settings.addCompatibility().doNotExpandShiftReturn();` but when I try to use this it says addCompatibility is not a function.

Kindly help me with this.

Thanks

**Comment by DRG-Support** [comment]
I have the same issue and just spent an hour looking through the source code and as far as I can tell its not possible to make this setting in any way. Will raise an issue to try and resolve.
